reactioncommerce/reaction
# perf: wrap tests in check to make sure app is ready when they run

Resolves #5315  
Impact: **minor**  
Type: **performance**

## Issue
CI `test-app` tests intermittently fail with a timeout, but usually pass on a second or third run.

## Solution
Wrap each test in `Reaction.onAppStartupComplete(() => { /* tests */ })` to make sure app is ready before tests run.

## Breaking changes
None

## Testing
1. run tests a multiple times
1. see that they pass every time
